WordPress/Classes/ViewRelated/NUX/CreateAccountAndBlogViewController.m

```diff
@@ -33,6 +37,7 @@ @interface CreateAccountAndBlogViewController ()<UITextFieldDelegate,UIGestureRe
     WPWalkthroughTextField *_emailField;
     WPWalkthroughTextField *_usernameField;
     WPWalkthroughTextField *_passwordField;
+    UIButton *_onePasswordButton;
```

Comment:
We should really refactor these ivars to props.  Just an observations, not a call to action.